packages/client/src/components/NavBar/index.js

```diff
@@ -51,85 +47,125 @@ const LinkStyled = styled(Link)`
   }
 `
 
-export default function NavBar(props)  {
-
-  const [anchorEl, setAnchorEl] = useState(null);
-  const open = Boolean(anchorEl);
-  let menuOpen = false;
+const UserLinkWrapper = styled.div`
+  margin-left: auto;
+`;
 
-  const handleMenu = (event) => {
-    setAnchorEl(event.currentTarget);
+export default function NavBar(props)  {
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+  const [menuDropdownAnchor, setMenuDropdownAnchor] = useState(null);
+  const handleMenuDropdownClick = (event) => {
+    setMenuDropdownAnchor(event.currentTarget);
+    event.stopPropagation();
   };
-
-  const handleClose = () => {
-    menuOpen = false;
-    setAnchorEl(null);
+  const handleMenuDropdownClose = () => {
+    setMenuDropdownAnchor(null);
   };
 
   let userLinks;
   if (true) {
     userLinks = (
       <>
-        <LinkStyled to='/login' ver='default'>Login</LinkStyled>
-        <div/>
-        <Modal title="Sign Up" trigger={<Button theme='accent' size='sm'> Sign Up </Button>}> <Wizard content={Wiz_content} /> </Modal>
+        <LinkStyled to='/signin' ver='default'>Login</LinkStyled>
+        <LinkStyled to='/signup' ver='default'>
+          <Button theme='accent' size='sm'>
+            Sign Up
+          </Button>
+        </LinkStyled>
       </>
     );
   } else {
-    userLinks = (
-      <>
-        <IconButton
-          aria-label="account of current user"
-          aria-controls="menu-navbar"
-          aria-haspopup="true"
-          onClick={handleMenu}
-          color="inherit"
-          onMouseOver={handleMenu}
-        >
-          <AccountCircleIcon/>
-          <div style={{ padding:'10px' }}/>
-          <LinkStyled ver='default' style={{ color: COLORS.blue }}>
-            {'Tim Beaver'}
-          </LinkStyled>
-        </IconButton>
-        <Menu
-          id="menu-navbar"
-          anchorEl={anchorEl}
-          getContentAnchorEl={null}
-          anchorOrigin={{
-            vertical: 'bottom',
-            horizontal: 'center',
-          }}
-          keepMounted
-          transformOrigin={{
-            vertical: 'top',
-            horizontal: 'center',
-          }}
-          open={open}
-          onClose={handleClose}
-          MenuListProps={{ onMouseLeave: handleClose }}
-        >
-          <MenuItem component={Link} to="/profile" style={{ fontSize: TextThemes.fontSize['default'] }}>Dashboard</MenuItem>
-          <MenuItem style={{ color: 'red', fontSize: TextThemes.fontSize['default'] }}>Sign Out</MenuItem>
-        </Menu>
-      </>
-    )
+    // userLinks = (
```

Comment:
Let's remove commented code when merging to the master branch. We can always look back at the branch's commit when trying to find the code that was commented.